docs/usage.mdx

```diff
@@ -18,7 +18,6 @@ Options:
       --usePolling        use polling for watch mode  [boolean] [default: false]
       --verbose           enable verbose logs  [boolean] [default: false]
       --noInclude         do not copy include files  [boolean] [default: false]
-      --logStringChanges  logs changes to legacy string argument offsets  [boolean] [default: false]
```

Comment:
I don't think this change belongs in this PR - but if we are updating this list, we should probably add the new options too (`writeTransformedFiles`, `optimizedLoops`, `allowCommentDirectives`).